docs/content/v0.20/architecture.md

```diff
@@ -62,21 +63,21 @@ to find objects that are synced by the OCM agent, gets their status
 and updates *WorkStatus* objects in the ITS namespace associated with the WEC.
 
 <figure>
-  <img src="./main-modules.png"  alt="Main Modules">
+  <img src="./images/main-modules.png"  alt="Main Modules">
   <figcaption align="center">Figure 2 - Main Modules </figcaption>
 </figure>
 
 ## KubeStellar Controller Manager
 
-This module manages the placement and status controllers. The placement controller watches 
-placement and workload objects on the Workload Definition Space 
-(WDS) and wraps workload objects into a manifest for delivery through the 
-Inventory and Transport Space (ITS). The status controller watches for *WorkStatus* objects on the 
-ITS and updates the status of objects in the WDS when singleton status is requested 
-in the placement for those objects. There is one instance of a KubeStellar Controller 
-Manager for each WDS. Currently this controller-manager runs in the KubeFlex hosting cluster and 
-is responsible of installing the required CRDs in the associated WDS.
-More details on the internals of this module are provided in [KubeStellar Controllers Architecture](#kubestellar-controllers-architecture)
+This module manages binding controller (a.k.a placement controller before the rename mentioned at the very beginning of this doc) and status controller. The binding controller watches `BindingPolicy` and workload objects on the Workload Definition Space (WDS) and wraps workload objects into a manifest for delivery through the Inventory and Transport Space (ITS). The status controller watches for *WorkStatus* objects on the ITS and updates the 
```

Comment:
The statement that this controller "wraps workload objects into **a** manifest" mixes English plural and singular in a way that is unclear in the presence of a general number of BindingPolicy objects. More explicit quantifiers/relations are needed. I suspect the intent is to describe the future in which only pluggable transport is operating. This doc also needs to be clear about the present, in which only the kslight POC behavior is operating.

Reply:
Also, the language "for delivery through the ... ITS" is not entirely clear about whether the wrapped manifests are already in the ITS or will go into them.  So I would reword further, perhaps something like the following.

> ... and maintains in the ITS a wrapped object per workload object to be delivered.

Reply:
fixed